README.md

```diff
@@ -97,13 +110,38 @@ make a tarball.
 
 To run it, create a file named `example.yaml` and run:
 
-    debos example.yaml
+    $ debos example.yaml
 
 The final tarball will be named "debian.tgz" if you would like to modify
 this name, you can provided a different name for the variable image like
 this:
 
-    debos -t image:"debian-arm64.tgz" example.yaml
+    $ debos -t image:"debian-arm64.tgz" example.yaml
+
+## Run example (using Docker)
+
+```$ sudo chmod 777 /dev/kvm```
+
+```$ sudo docker run --rm --interactive --tty --device /dev/kvm --user $(id -u) --workdir /recipes --mount "type=bind,source=$(pwd),destination=/recipes" --security-opt label=disable <debos-docker-image-id> example.yaml ```
+
+or run:
```

Comment:
i don't think these scripts add much value tbh

Reply:
Ok. I test debos fs on imx8 :)

[seco-c61]( https://www.seco.com/it/products/sbc-c61)

Except tty configurations all seems work :) I'm in.

The log:

```
U-Boot 2019.04-gb13fa9c-dirty (Oct 26 2020 - 17:51:30 +0100)

CPU:   Freescale i.MX8MMQ rev1.0 1600 MHz (running at 1200 MHz)
CPU:   Industrial temperature grade (-40C to 105C) at 48C
Reset cause: POR
Model: SECO i.MX8MM C61 board
DRAM:  4 GiB
DEBUG! board_init, 377 BOOT FROM ----> 7 
MMC:   KERN_ERR, DEBUG FUNC = fsl_esdhc_init, LINE = 1250, 
KERN_ERR, DEBUG FUNC = fsl_esdhc_init, LINE = 1250, 
FSL_SDHC: 1, FSL_SDHC: 2
Loading Environment from MMC... Run CMD11 1.8V switch
OK
System data from MMC1... OK.
In:    serial
Out:   serial
Err:   serial

 BuildInfo:
  - ATF a-20190
  - U-Boot 2019.04-gb13fa9c-dirty

flash target is MMC:1
Run CMD11 1.8V switch
Net:   
Warning: ethernet@30be0000 (eth0) using random MAC address - ee:6a:e7:4d:9e:67
eth0: ethernet@30be0000
Fastboot: Normal
Normal Boot
Hit any key to stop autoboot:  0 
u-boot seco c61=> 
u-boot seco c61=> boot
Run CMD11 1.8V switch
switch to partit Run CMD11 1.8V switch
==> Running in Normal Mode...
Running in Normal Mode...
Run CMD11 1.8V switch
switch to partit 24891904 bytes read in 317 ms (74.9 MiB/s)
Run CMD11 1.8V switch
switch to part 57682 bytes read in 13 ms (4.2 MiB/s)
Running in Normal Mode...
## Flattened Device Tree blob at 43000000
   Booting using the fdt blob at 0x43000000
   Loading Device Tree to 00000000fe90f000, end 00000000fe920151 ... OK

Starting kernel ...

[    0.000000] Booting Linux on physical CPU 0x0000000000 [0x410fd034]
[    0.000000] Linux version 4.19.35-g8edaeca-dirty (tom@tom-desktop) (gcc version 8.3.0 (GCC)) #2 SMP PREEMPT Tue Oct 27 11:35:17 CET 2020
[    0.000000] Machine model: FSL i.MX8MM C61 board


# apt
apt         apt-cdrom   apt-get     apt-mark    
apt-cache   apt-config  apt-key

# ls
bin   dev  home  lost+found  mnt  proc  run   srv  tmp  var
boot  etc  lib   media       opt  root  sbin  sys  usr

```